stack/rest/src/test/java/org/apache/usergrid/rest/AbstractRestIT.java

```diff
@@ -39,23 +39,33 @@
 import org.apache.usergrid.java.client.Client;
 import static org.apache.usergrid.utils.JsonUtils.mapToFormattedJsonString;
 import static org.apache.usergrid.utils.MapUtils.hashMap;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.jboss.shrinkwrap.resolver.api.maven.archive.importer.MavenImporter;
 import org.junit.AfterClass;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 import org.junit.Before;
 import org.junit.ClassRule;
+import org.junit.runner.RunWith;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 
 /**
- * Base class for testing Usergrid Jersey-based REST API. Implementations should model the 
- * paths mapped, not the method names. For example, to test the the "password" mapping on 
- * applications.users.UserResource for a PUT method, the test method(s) should following the 
+ * Base class for testing Usergrid Jersey-based REST API. Implementations should model the
+ * paths mapped, not the method names. For example, to test the the "password" mapping on
+ * applications.users.UserResource for a PUT method, the test method(s) should following the
  * following naming convention: test_[HTTP verb]_[action mapping]_[ok|fail][_[specific
  * failure condition if multiple]
  */
-//@Concurrent()
+//
+@RunWith( Arquillian.class )
 public abstract class AbstractRestIT extends JerseyTest {
```

Comment:
Yeah, just the @RunwithAttribute and the static method telling it how to create the testable archive.  Everything else is encapsulated within the arquillian.xml file